platform/FileHandle.h

```diff
@@ -18,9 +18,17 @@
 
 typedef int FILEHANDLE;
 
-#include <stdio.h>
+#include <cstdio>
+#include "Callback.h"
 #include "platform/platform.h"
 
+#define MBED_POLLIN         0x0001
+#define MBED_POLLOUT        0x0010
+
+#define MBED_POLLERR        0x1000
+#define MBED_POLLHUP        0x2000
+#define MBED_POLLNVAL       0x4000
```

Comment:
I think so, yes. I initially assumed no, but having seen what we're doing with errno, it would fit.